src/components/Button.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './styles/Button.css';
+
+const isOperator = val => ['÷', 'x', '-', '+', '='].includes(val);
+
+const isZero = val => val === '0';
+
+const Button = ({name, clickHandler}) => {
+    const handleClick = () => clickHandler(name);
+    return (
+    <div 
+    className={`button-wrapper ${
```

Comment:
It would be cleaner if we just declare a variable for this className:

```javascript
const operatorClass = isOperator(name) ? 'operator' : ''; // we never should use `null` as a className

const zeroClass = isZero(name) ? 'zero' : ''; 
```

Now, in the button's classname:

```javascript
<div
 className = {`button-wrapper ${operatorClass} ${zeroClass}`}
```